pootle/apps/accounts/adapter.py

```diff
@@ -47,3 +48,11 @@ def is_open_for_signup(self, request):
     def add_message(self, request, level, message_template, *args, **kwargs):
         """Silence messages altogether."""
         pass
+
+    def send_confirmation_mail(self, *args, **kwargs):
+        try:
+            super(PootleAccountAdapter, self).send_confirmation_mail(*args,
+                                                                     **kwargs)
+        except Exception:
+            raise Exception(_('Some problem happened when tried to send the '
+                              'confirmation email. Please try again later.'))
```

Comment:
Could we capture and log the exception - so sysadmins can find out that its happened